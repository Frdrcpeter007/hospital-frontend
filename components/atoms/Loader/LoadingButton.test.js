import LoadingButton from './LoadingButton';
import { render } from '@testing-library/react';

describe('LoadingButton', () => {
  it('renders', () => {
    const { container } = render(<LoadingButton />);
    expect(container).toMatchSnapshot();
  });
});
