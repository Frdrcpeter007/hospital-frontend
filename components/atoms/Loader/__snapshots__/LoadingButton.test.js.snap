// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingButton renders 1`] = `
<div>
  <div
    class="flex gap-2 items-center justify-center"
  >
    <div
      arialabel="loading"
      class="animate-spin inline-block w-4 h-4 border-[2px] border-current border-t-transparent text-white rounded-full"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
    <span>
      signIn.buttons.waiting
    </span>
  </div>
</div>
`;
